### ISSUE TITLE
Works, but raises SyntaxError with JRuby 9.2.19.0 and OpenJDK

### ISSUE THREAD

**Issue by todd-a-jacobs** [ruby-next/ruby-next#81]
## What did you do?

1. Installed JRuby-9.2.19.0
2. Installed Ruby-Next v0.13.1
3. Ran the same endless def you have in the [quickstart section](https://github.com/ruby-next/ruby-next#quick-start) of your README.

## What did you expect to happen?
For the code to print `👽` without errors.

## What actually happened?
The code appeared to execute properly, in the sense that the endless def printed the correct value, but JRuby still raised:

``` ruby
SyntaxError: -e:2: syntax error, unexpected '='
def greet(val) =
               ^
👽
```

## Environment
`jruby --version`
> jruby 9.2.19.0 (2.5.8) 2021-06-15 55810c552b OpenJDK 64-Bit Server VM 17+0 on 17+0 +jit [darwin-x86_64]

`jruby -e 'puts RUBY_VERSION'`
> 2.5.8

`gem list ruby-next`

>     *** LOCAL GEMS ***
>
>     ruby-next (0.13.1)
>     ruby-next-core (0.13.1)
>     ruby-next-parser (3.0.1.0)

`gem list unparser`

>     *** LOCAL GEMS ***
>
>     unparser (0.6.0)

`java --version`
>     openjdk 17 2021-09-14
>     OpenJDK Runtime Environment Homebrew (build 17+0)
>     OpenJDK 64-Bit Server VM Homebrew (build 17+0, mixed mode, sharing)

`uname -v`
>     Darwin Kernel Version 20.6.0: Mon Aug 30 06:12:21 PDT 2021; root:xnu-7195.141.6~3/RELEASE_X86_64

**Comment by palkan** [comment]
Thanks!

Looks like a JRuby problem, opened an issue: https://github.com/jruby/jruby/issues/6874

**Comment by palkan** [comment]
Fixed in JRuby 9.3.1.0
